cloudformation/fargate.private.yaml

```diff
@@ -210,7 +210,7 @@ Resources:
       AllocatedStorage: "80"
       DBInstanceClass: "db.m4.large"
       Engine: postgres
-      EngineVersion: "10.6"
+      EngineVersion: "11.12"
```

Comment:
as of july 12, 2021: https://aws.amazon.com/about-aws/whats-new/2021/07/amazon-rds-for-postgresql-supports-new-minor-versions/